.pre-commit-config.yaml

```diff
@@ -0,0 +1,12 @@
+repos:
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v2.31.0
+    hooks:
+      - id: pyupgrade
+        args:
+          - --py38-plus
```

Comment:
I'm kind of of the opinion the more the merrier with pre-commit hooks and this one was easy to add.

Reply:
+1. We're also already using this pre-commit in `dask/dask` and `dask/distributed` -- so I like the consistent code styling

Reply:
right. We can add black and things when we are feeling more ambitious :)